source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -516,6 +530,22 @@ const Network::Address::Instance& HttpConnectionManagerConfig::localAddress() {
   return *context_.localInfo().address();
 }
 
+/**
+ * Determines what tracing provider to use for a given `envoy.http_connection_manager`
+ * filter instance.
+ */
+const envoy::config::trace::v3::Tracing_Http* HttpConnectionManagerConfig::getPerFilterTracerConfig(
+    const envoy::extensions::filters::network::http_connection_manager::v3::
+        HttpConnectionManager&) {
+  // at the moment, it is not yet possible to define tracing provider as part of
+  // `envoy.http_connection_manager` config.
```

Comment:
nit: same name nit as above (there are a few other places, please grep for them all), also start sentence with capital A

Reply:
fixed